src/content/1.7/modules/core-updates/1.7.8.md

```diff
@@ -0,0 +1,149 @@
+| composer/package-versions-deprecated | NEW     | 1.11.99 |                                                                                     |
+| prestashop/blockwishlist             | NEW     | v2.0.0  |                                                                                     |
+| symfony/polyfill-intl-normalizer     | NEW     | v1.18.1 |                                                                                     |
+
+| Dev Changes                     | From    | To      | Compare                                                                        |
+|---------------------------------|---------|---------|--------------------------------------------------------------------------------|
+| composer/xdebug-handler         | 1.4.0   | 1.4.3   | [...](https://github.com/composer/xdebug-handler/compare/1.4.0...1.4.3)        |
+| friendsofphp/php-cs-fixer       | v2.16.4 | v2.17.2 | [...](https://github.com/FriendsOfPHP/PHP-CS-Fixer/compare/v2.16.4...v2.17.2)  |
+| johnkary/phpunit-speedtrap      | v3.1.0  | v3.2.0  | [...](https://github.com/johnkary/phpunit-speedtrap/compare/v3.1.0...v3.2.0)   |
+| myclabs/deep-copy               | 1.9.4   | 1.10.1  | [...](https://github.com/myclabs/DeepCopy/compare/1.9.4...1.10.1)              |
+| phake/phake                     | v3.1.7  | v3.1.8  | [...](https://github.com/phake/Phake/compare/v3.1.7...v3.1.8)                  |
+| phpdocumentor/reflection-common | 2.0.0   | 2.1.0   | [...](https://github.com/phpDocumentor/ReflectionCommon/compare/2.0.0...2.1.0) |
+| phpspec/prophecy                | 1.10.1  | v1.10.3 | [...](https://github.com/phpspec/prophecy/compare/1.10.1...v1.10.3)            |
+| symfony/phpunit-bridge          | v3.4.36 | v3.4.43 | [...](https://github.com/symfony/phpunit-bridge/compare/v3.4.36...v3.4.43)     |
+| webmozart/assert                | 1.6.0   | 1.9.1   | [...](https://github.com/webmozarts/assert/compare/1.6.0...1.9.1)              |
+
+### Packages added
+
+| Production Changes                   | From    | To      |
+|--------------------------------------|---------|---------|
+| composer/package-versions-deprecated | NEW     | 1.11.99 |
+| prestashop/blockwishlist             | NEW     | v2.0.0  |
+| symfony/polyfill-intl-normalizer     | NEW     | v1.18.1 |
+
+| Dev Changes                     | From    | To      |
+|---------------------------------|---------|---------|
+| phpstan/phpstan                 | NEW     | 0.12.80 |
+| prestashop/phpstan-prestashop   | NEW     | 1.1.1   |
+
+
+### Packages removed
+
+| Production Changes                   | From    | To      |
+|--------------------------------------|---------|---------|
+| prestashop/sekeywords                | v2.0.0  | REMOVED |
+| prestashop/statsequipment            | v2.0.0  | REMOVED |
+| prestashop/statslive                 | v2.1.0  | REMOVED |
+| prestashop/statsorigin               | v2.0.2  | REMOVED |
+| prestashop/statsvisits               | v2.0.3  | REMOVED |
```

Comment:
No need to add "To" since it's been removed!
```suggestion
| Package name                         | Version |
|--------------------------------------|---------|
| prestashop/sekeywords                | v2.0.0  |
| prestashop/statsequipment            | v2.0.0  |
| prestashop/statslive                 | v2.1.0  |
| prestashop/statsorigin               | v2.0.2  |
| prestashop/statsvisits               | v2.0.3  |
```